### PR TITLE
Remove closed offices from Careers page (fixes #12270)

### LINKED ISSUE #12270: Update office locations in Careers pages
## Description

Some information on the Careers pages are not up-to-date regarding dedicated Mozilla office locations. It would be best to remove them to keep the pages relevant and clear.

The office location information in position pages also needs to be updated, as we have all the former office locations still listed.

#### Relevant URLs:
- https://www.mozilla.org/en-US/careers/
- https://www.mozilla.org/en-US/contact/spaces/
- https://www.mozilla.org/en-US/careers/position/gh/4457087/ (scroll to bottom to 'Our Impact is Global' headline)

---

#### Success Criteria
- [ ] Remove closed offices from Careers homepage
- [ ] Remove closed offices from `/contact/spaces` page
- [ ] Remove closed offices from Position pages

### PR DESCRIPTION
**Pull request by reemhamz**
## One-line summary
Removing closed office information from Careers pages. 
Primarily removed mentions of the Portland office, as well as the count of offices within the Positions pages

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/12270

## Testing
Run `y.test -r a tests/redirects` for the updated old `contact/spaces` redirect

### To test this work:

- [x] http://localhost:8000/en-US/contact/spaces/ (removed Portland office from sidebar)
- [x] http://localhost:8000/en-US/careers/ (removed Portland office from 'A global view' section)
- [x] http://localhost:8000/en-US/careers/position/gh/3920659/ (removed old office locations and updated the math under 'Our impact is global' headline)

### CODE REVIEW

**Review by craigcook — CHANGES_REQUESTED** [review]
LGTM, just one suggestion. 

And I see "Portland Office" in the Locations dropdown on the job listings page but I think that needs to be removed in GlassDoor, I don't think it lives in bedrock anywhere that I can find. Not sure who we need to talk to about that.

File: bedrock/mozorg/urls.py [hunk 1]
```diff
@@ -88,7 +88,6 @@
     page("contact/spaces/beijing/", "mozorg/contact/spaces/beijing.html"),
     page("contact/spaces/berlin/", "mozorg/contact/spaces/berlin.html"),
     page("contact/spaces/paris/", "mozorg/contact/spaces/paris.html"),
-    page("contact/spaces/portland/", "mozorg/contact/spaces/portland.html"),
```
[the remark this answers is not in the document]
Comment (reemhamz) [reply bedrock/mozorg/urls.py:91]: nice catch!

**Review by craigcook — APPROVED** [review]
🏢

**Comment by craigcook** [comment]
D'oh, I missed this test: https://github.com/mozilla/bedrock/blob/main/tests/functional/test_contact.py#L22 I'll submit a PR to fix it.

**Pull request merged**